### PR TITLE
Fix Bug 1658867: Fix for tags in the filter panel 

### PR DESCRIPTION
**Pull request by Mitch4sho**
Addressing Bug 1658867 Tag priority. Fix for tags in the filter panel to put in order of highest priority to lowest, Changes tested by flow.

### CODE REVIEW

**Comment by abowler2** [comment]
Hey @Mitch4sho, congrats on your first PR for Pontoon! 🎉 

I noticed that your Prettier settings made changes that aren't consistent with the settings for Pontoon. You can run `make prettier` on your branch to have the correct formatting applied and then push those changes (you might need to disable your editor from formatting on save in order to not have those changes overridden).

Let me know if you need any help with that.

File: frontend/src/modules/search/components/FiltersPanel.js [hunk 1]
```diff
@@ -298,53 +298,60 @@ export class FiltersPanelBase extends React.Component<Props, State> {
...
-                                                onClick={this.createApplySingleFilter(
-                                                    tag.slug,
-                                                    'tags',
-                                                )}
-                                            >
-                                                <span
-                                                    className='status fa'
-                                                    onClick={this.createToggleFilter(
+                                    {this.props.tagsData
+                                        .sort((a: Tag, b: Tag) => {
```
Comment (abowler2) [inline frontend/src/modules/search/components/FiltersPanel.js:302]: From working with you on this I know that the `Tag` types were added in the attempt to work out what was causing Flow errors. I'm not sure if they are needed here now and will defer to @mathjazz on that.

**Review by abowler2 — COMMENTED** [review]
Nice work!  

This sorts the tags as expected, and pending the below comment re: typing the arguments in Sort, the code looks good to me too.

File: frontend/src/modules/search/components/FiltersPanel.js (hunk 1 above)
Comment (mathjazz) [reply frontend/src/modules/search/components/FiltersPanel.js:302]: Yeah, please remove the `Tag` types - the rule of the thumb is not to use them as long as Flow doesn't complain.

Please also replace `this.props.tagsData` with `props.tagsData`.

There's a lot of indent in here. One day we should refactor and probably move each Filter section into a separate component.

Comment (Mitch4sho) [reply frontend/src/modules/search/components/FiltersPanel.js:302]: Okay, will remove the `Tag` types, replace with `props.tagData`and recommit the following. Thank you for reviewing!

**Review by mathjazz — APPROVED** [review]
Nice work!

Thanks for the patch. And welcome to Pontoon! 🥳

**Comment by Mitch4sho** [comment]
Thank you!

**Comment by mathjazz** [comment]
Works beautifully on stage:
https://mozilla-pontoon-staging.herokuapp.com/sl/firefox/all-resources/?string=74127

**Pull request merged**